pom.xml

```diff
@@ -118,7 +118,7 @@
 
     <sonar.organization>samply</sonar.organization>
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-    <slf4jVersion>1.7.30</slf4jVersion>
```

Comment:
Please look at [https://www.slf4j.org/manual.html](url). We want to use slf4j and log4j as the backend. In the future we can then decide if we want to change to logback or something else.

Reply:
Please move the property to the position, the log4j version was before.